tests/dummy/app/pods/view/renderers/views/checkbox-array.js

```diff
@@ -3,12 +3,54 @@ export default {
     main: {
       children: [
         {
-          model: 'severity',
-          renderer: {
-            name: 'checkbox-array'
-          }
+          children: [
+            {
+              model: 'severity',
+              renderer: {
+                name: 'checkbox-array'
+              }
+            }
+          ],
+          label: 'Severity',
+          collapsible: true
+        },
+        {
+          children: [
+            {
+              model: 'serviceAffecting',
+              renderer: {
+                name: 'checkbox-array',
+                data: [
```

Comment:
I had done it this way originally. @sandersky had said he thought it was overly complicated and preferred to simply override the enum completely. But if you both agree we can look at it agian.